src/core/AnimatedNode.js

```diff
@@ -52,9 +62,14 @@ function runPropUpdates() {
   loopID += 1;
 }
 
-let nodeCount = 0;
-
 export default class AnimatedNode {
+
+  /* { [loop: string]: int } */
```

Comment:
nit: this comment does not tell me anything. Can you use some words if you want to explain it? If not, just drop